modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/util/WorkflowTaskTestUtil.java

```diff
@@ -30,21 +30,29 @@
 	public static WorkflowTask getWorkflowTask(long workflowInstanceId)
```

Comment:
Make sure which this method is only used in WorkflowTaskResourceTest, maybe, it is breaking other test classes.